scripts/push-schema-changes.js

```diff
@@ -34,30 +34,32 @@ async function updateSchemaFile({
     assignees: ["artsyit"],
     labels: ["Squash On Green"],
     update: (repoDir) => {
-      const repoDest = path.join(repoDir, dest)
-      execSync("yarn config set ignore-engines true", { cwd: repoDir })
-      execSync("yarn install", { cwd: repoDir })
-      if (dest.endsWith(".json")) {
-        const sdl = readFileSync("_schemaV2.graphql", "utf8").toString()
-        const schema = buildSchema(sdl, { commentDescriptions: true })
-        const gql = graphqlSync(schema, introspectionQuery)
-        writeFileSync(repoDest, JSON.stringify(gql, null, 2))
-      } else {
-        execSync(`cp _schemaV2.graphql '${repoDest}'`)
+      destinations.forEach((dest) => {
+        const repoDest = path.join(repoDir, dest)
+        execSync("yarn config set ignore-engines true", { cwd: repoDir })
```

Comment:
These two steps can probably go a few lines above _outside_ the `forEach` loop (since we know we want to always run this once per repo).

Reply:
Ah yes, nice catch 👍 fixed